home/utils.py

```diff
@@ -306,20 +359,41 @@ def get_side_panel_width(ws, column_heading):
                 break
             if next_row["whatsapp_body"] not in ["", None]:
                 whatsapp_messages.append(next_row["whatsapp_body"])
+            if next_row["variation_body"] not in ["", None]:
+                variation_messages.append(
+                    {
+                        "variation_body": next_row["variation_body"],
+                        "variation_title": next_row["variation_title"],
+                        "message_number": next_row["message"],
+                    }
+                )
+
             if next_row["messenger_body"] not in ["", None]:
                 messenger_messages.append(next_row["messenger_body"])
             if next_row["viber_body"] not in ["", None]:
                 viber_messages.append(next_row["viber_body"])
-
-        contentpage = add_web(row)
-        contentpage = add_whatsapp(row, whatsapp_messages, contentpage)
-        contentpage = add_messenger(row, messenger_messages, contentpage)
-        contentpage = add_viber(row, viber_messages, contentpage)
+        exiting_contentpage = ContentPage.objects.filter(pk=pk).first()
```

Comment:
What if the existing content page is a completely different page? Will the current content be overwritten?
Eg we decide we want YAL to have a section for teen mothers so we import content from momconnect, page with id 3  in the momconnect data will be very different to the page with id 3 in the YAL data.

Reply:
Yeah this is the case, but SXD wants to be able to set the page ID's and I don't want to check things on title's because with projects like OOY the titles changed a few times. I will speak to Helen and find out what she'd like but I think the best might be to add `page_id`'s to the contentpage view and allow the admin to sort and search on page id or something